README.md

```diff
@@ -1,27 +1,148 @@
-# Role Name
+# Gitlab
 
-A brief description of the role goes here.
+This role configures and installs a gitlab instance
 
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
+This role depends on postgres, redis and nginx
```

Comment:
You could link to the exact roles.

Reply:
Link the roles. Also, they are called PostgreSQL and Redis